nautobot/extras/api/relationships.py

```diff
@@ -140,44 +144,93 @@ def to_representation(self, value):
                         {"id": other_obj.id} for other_obj in other_objects
                     ]
 
-        logger.info("to_representation(%s) -> %s", value, data)
+        logger.debug("to_representation(%s) -> %s", value, data)
         return data
 
     def to_internal_value(self, data):
         """
         Allow for updates to some relationships without overriding others.
+
+        Returns:
+            <self.field_name>: {
+                <relationship>: {
+                    "source": [<object>, <object>,...]
+                },
+                <relationship>: {
+                    "destination": [<object>, <object>,...]
+                },
+                <relationship>: {
+                    "destination": [<object>, ...],
+                    "source": [...],
+                },
+                <relationship>: {
+                    "peer": [<object>, <object>, ...],
+                },
```

Comment:
I know this is an advanced feature so it might not be necessary to change but it's not immediately apparent whether this method returns all of these keys, only one or a combination of keys.

Reply:
t will be one of these four cases for each relationship - either `source` alone, `destination` alone, both `source` and `destination`, or `peer` alone. Any thoughts how I can make it more clear?

Reply:
It makes sense in the context of custom relationships and since this is specific to that feature it's probably fine as-is